app/Models/EventLog.php

```diff
@@ -98,6 +99,26 @@ class EventLog extends Model
         'their_userdata' => 'JsonValue',
     ];
 
+    public static function fetchUnpaidEvents(): \Illuminate\Support\Collection
```

Comment:
Return type could be `Collection`, because it is Eloquent

Reply:
+1